src/stem/settings.py

```diff
@@ -139,10 +139,47 @@
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 
 # API
-REST_FRAMEWORK = {
+#REST_FRAMEWORK = {
```

Comment:
I would go ahead and clean up all the commented pieces of they’re no longer needed